frontend/src/Wanted/CutoffUnmet/CutoffUnmet.js

```diff
@@ -252,7 +252,7 @@ class CutoffUnmet extends Component {
                         Are you sure you want to search for all {totalRecords} Cutoff Unmet episodes?
                       </div>
                       <div>
-                        This cannot be cancelled once started without restarting Sonarr.
+                        This cannot be cancelled once started without disabling all of your indexers.
```

Comment:
`This cannot be cancelled once started without restarting Sonarr or disabling all of your indexers.`